docs/source/reference/faq.md

```diff
@@ -1,5 +1,50 @@
 # FAQ
 
+## How to resume a failed release?
+
+If the workflow failed to publish the release, for example because PyPI or npm was down during the release process, we can try to re-run the failed "Publish Release" workflow.
+
+If the draft GitHub release was correctly created, re-run the workflow this time specifying `populate-release` as a step to skip. That way the assets already attached to the draft GitHub Release and the associated release commit will not be recreated, and the workflow will skip to the "Finalize Release" step directly.
+
+## Failed to publish a package to `npm`
+
+The releaser may fail to publish a package to the `npm` in the following cases:
+
+- `npmjs.com` is down, or `npm` is encountering issues publishing new packages
```

Comment:
I was thinking whether to add a point about some other once we saw in JupyterLab:
- Wrong URLs break publishing to npm with provenance: https://github.com/jupyterlab/jupyterlab/pull/15462
- Release team missing privileges on packages removed in 4.0: https://github.com/jupyterlab/jupyterlab/issues/15677

but I am not sure how often these pop up.

Reply:
> This one should normally be caught by check-release?

Yes, that's what I was thinking